docs/sceneeditorwidget.rst

```diff
@@ -7,20 +10,26 @@ The **Scene Editor Widget** is a control to add, remove, and modify the graphics
 
 .. figure:: _images/scene-editor-widget.png
    :alt: Scene editor widget.
+   :width: 75%
 
    Scene editor widget.
 
 Graphics list
 -------------
 
+Along the top of the scene editor window is a region selector that allows you to select the region of current scene.
+
+Below the selector is the list of currently defined graphics.
 The graphics list displays all the defined graphics in the current scene, each with a checkbox in front of it. 
 This checkbox allows you to set whether to display this graphic.
-Above the chart list is a region selector that allows you to select the region of current scene.
-Below the list of graphics, there is a drop-down menu that allows you to add a new graphic.
-There is also a button that allows you to delete the selected graph.
+This graphics list can also be reordered via drag and drop. Graphics at the top have higher display priority.
```

Comment:
Earlier graphics are rendered first, which can affect translucency results in some modes.